.github/workflows/php.yaml

```diff
@@ -48,17 +55,30 @@ jobs:
   codestyle:
       name: "Code style and static analysis"
       runs-on: ubuntu-latest
+
+      strategy:
```

Comment:
Employing a build matrix here so we can reuse the hash key configuration.

Does that make sense?